__tests__/commands/install/integration.js

```diff
@@ -152,6 +152,31 @@ test.concurrent(
   },
 );
 
+test.concurrent('replace the symlink when it changes, when using the link: protocol', async () => {
+  await runInstall({}, 'install-link', async (config, reporter): Promise<void> => {
+    const lockfile = await Lockfile.fromDirectory(config.cwd);
+
+    const pkgJson = await fs.readJson(`${config.cwd}/package.json`);
+    pkgJson.dependencies['test-missing'] = 'link:barbaz';
+    await fs.writeFile(`${config.cwd}/package.json`, JSON.stringify(pkgJson));
+
+    const install2 = new Install({}, config, reporter, lockfile);
```

Comment:
Okay, how about `secondInstall` or `secondRound` for a more descriptive name then?